client/src/components/nav/Navigation.tsx

```diff
@@ -35,3 +47,44 @@ export const Navigation = () => {
     </div>
   );
 };
+
+const LogoBanner = styled.div`
+  display: flex;
+  width: 100%;
+  background-color: #F5F5F5;
+  justify-content: space-between;
+`;
+
+const Logo = styled.div`
+  align-self: flex-start;
+`;
+
+const TitleText = styled.div`
+  display: flex;
+  margin: auto;
+  align-self: center;
+  font-size: 22px;
+`;
+
+const NavBar = styled.div`
```

Comment:
and you can change this to styled.nav

and add
position: sticky;
top: 0;

to have a sticky navbar